src/pages/likes/likes.js

```diff
@@ -0,0 +1,38 @@
+import { React, useEffect } from "react";
+import "./likes.css";
+import { useNavigate } from "react-router";
+import { ActionVideoCard } from "../../components/card/action-video-card";
+import { useAuth, useLikedVideos } from "../../context";
+
+function Likes() {
+    const { likedVideos, dispatchLikedVideos, removeFromLikedVideosHandler } =
+        useLikedVideos();
+    const { itemInLikedVideos } = likedVideos;
+
+    const { auth } = useAuth();
+    const navigation = useNavigate();
+
+    useEffect(() => {
+        auth.isAuthorized || navigation("/login");
```

Comment:
rather then using `useEffect` you can use `private routes` here

Reply:
I will do it in upcoming PRs